### ISSUE TITLE
Relations sync

### ISSUE THREAD

**Issue by Clovis-krz** [pocketbase/pocketbase#2583]
Hi,
Would it be possible to implement the synchronization of relations ? I will explain myself with an example, lets say i have a collection categories( id, name, relation(topics, multiple) ) and a collection topics( id, title, description, relation( categories, single ) ). If i do that currently then when i create a category and a topic, if i link from the category the topic, the category wont be mapped in topics and i will have to also add the category in the topic manually. It is not very convenient.
Basically, for each relation mapping we can check the type (single or multiple) and update the other collection accordingly, add to the list if its multiple or overwrite if its single.
I am aware of indirect expand though because it is only supporting single relation, it makes it useless in my example.
I love the project and hope the best !

**Comment by ganigeorgiev** [comment]
If I'm understanding your use case correctly, I don't think it will be a good idea as it will encourage users to have denormalized data (aka. having the same relation in multiple places). I understand that this may be desired in some situations but at least for now I don't plan working on this.

Support for indirect expand of multiple relation fields will be added soon.

**Comment by ganigeorgiev** [comment]
One eventual alternative in your case could be to have a "junction" collection that will hold only 2 `relation` fields pointing back to `topics` and `categories`.
